src/jit/morph.cpp

```diff
@@ -16209,6 +16209,23 @@ GenTreePtr Compiler::fgInitThisClass()
     }
     else
     {
+#ifdef FEATURE_READYTORUN_COMPILER
+        // Only CoreRT understands CORINFO_HELP_READYTORUN_GENERIC_STATIC_BASE. Don't do this on CoreCLR.
+        if (opts.IsReadyToRun() && eeGetEEInfo()->targetAbi == CORINFO_CORERT_ABI)
+        {
+            CORINFO_RESOLVED_TOKEN resolvedToken;
+            ZeroMemory(&resolvedToken, sizeof(resolvedToken));
```

Comment:
I copypasted this from the nongeneric ReadyToRun path. I will fix it in both in a separate pull request. I really want to merge this before the "#7837 Introduce new CORJIT_FLAGS type" pull request gets merged because I need this to unblock myself on CoreRT and I would rather not deal with that right now.